sonic_platform_base/fan_base.py

```diff
@@ -18,6 +18,7 @@ class FanBase(device_base.DeviceBase):
     # Possible fan directions (relative to port-side of device)
     FAN_DIRECTION_INTAKE = "intake"
     FAN_DIRECTION_EXHAUST = "exhaust"
+    FAN_DIRECTION_NOT_APPLICABLE = "not applicable"
```

Comment:
Maybe make the string `N/A` to be more brief?

Reply:
> Maybe make the string `N/A` to be more brief?

Updated.